doc/integrator/cluster.rst

```diff
@@ -0,0 +1,23 @@
+.. _integrator_cluster:
+
+Create a cluster
+================
+
+The first think to do to create cluster is to separate the database and the application.
+
+One possibility is to have the same database for all the chains (integration, production).
+An than we can have the same schema for the static data, and on a deployement can copy the 
+main schema (`Clone schema <https://wiki.postgresql.org/wiki/Clone_schema>`_).
```

Comment:
is ti better ? :
Then we can have the same schema for the static data, and on a deployment can copy the    
main schema (`Clone schema <https://wiki.postgresql.org/wiki/Clone_schema>`_)
from the integration to the production.